### DISCUSSION TITLE
How to get the RAW Data from CAN ID

### DISCUSSION THREAD

**Discussion by martin8310** [meatpiHQ/wican-fw#213] [Q&A]
Hello, 

Can someone explain to me how I can send the RAW data from an address via MQTT?

I would like to have this monitor data sent directly via MQTT. How does that work?
```
ID	Type	Len		Data				Time	Count
5D7h	Std	8		00 00 02 4B 1C 20 FA 00		108	189
```

I need the DATA block so that I can use this formula in Node-RED:
(B2 * 16.777.216 + B3 * 65536 + B4 * 256 + B5 ) / 16 / 100
Wican has no space for this under Expression.

regards Martin

**Comment by meatpiHQ** [comment]
@martin8310 this was answered on discord 

`[B2:B5]*6.25`
